src/Menu/Matcher/Voter/AdminVoter.php

```diff
@@ -36,7 +36,7 @@ class AdminVoter implements VoterInterface
     /**
      * @var Request
```

Comment:
Shouldn't the typehint be Request|null ?

Reply:
Maybe. Let me do some tests.

Reply:
Done.